data/categories.txt

```diff
@@ -3652,7 +3653,7 @@ pl:3^Szopa|schronienie
 pt:^Abrigo
 pt-BR:^Abrigo
 ro:^Adăpost
-es:^Refugio|Abrigo|Marquesina
+es:^Refugio|Albergue
```

Comment:
Are you sure that "Albergue" is correct?
See http://wiki.openstreetmap.org/wiki/Tag:amenity%3Dshelter